compiler/tests-common/tests/org/jetbrains/kotlin/codegen/AbstractLineNumberTest.java

```diff
@@ -58,7 +58,7 @@ private KotlinCoreEnvironment createEnvironment() {
     public void setUp() throws Exception {
         super.setUp();
 
-        KotlinCoreEnvironment environment = createEnvironment();
+        KotlinCoreEnvironment environment = createEnvironment(true);
```

Comment:
Why not compile this with ir backend?

Reply:
The errors I am getting during compilation look like:

```
UNRESOLVED_REFERENCE: Unresolved reference: lineNumber (2,21) in /class.kt
java.lang.IllegalStateException: UNRESOLVED_REFERENCE: Unresolved reference: lineNumber (2,21) in /class.kt
```

I read it as: currently ir backend does not generate kotlin.Metadata annotation and the generated class is not properly loaded.

Reply:
I should be redundant after rebase, it's only one compilation

Reply:
Right, it is gone after rebase